components/spinner/index.js

```diff
@@ -1,3 +1,10 @@
-const spinner = <span className="spinner is-active" />;
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
 
-export default () => spinner;
+function Spinner( { className } ) {
```

Comment:
You're not / no longer taking advantage of these enhancements? Should we omit them respecting our good friend [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it)?

Reply:
👍 I've removed these changes.